crates/full-node/sov-db/src/lib.rs

```diff
@@ -35,7 +35,7 @@ pub mod accessory_db;
 pub mod namespaces;
 
 /// Implements commit flag logic for state_db_nomt.
-pub(crate) mod commit_flag;
+pub mod commit_flag;
```

Comment:
Why should it become public? if it is only for `sov-test-utils`, then let's make it public only if `test-utils` is enabled. 

I don't want to expose too much of internals outside of the crate

Reply:
fixed